frontend/src/Editor/AppVersionsManager/ReleasedVersionError.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import cx from 'classnames';
+
+export const ReleasedVersionError = ({ isUserEditingTheVersion, changeBackTheState }) => {
+  React.useState(() => {
+    setInterval(() => changeBackTheState(), 2000);
+  }, [isUserEditingTheVersion]);
```

Comment:
Clear the `setInterval` when the component is unmounted. Even if we go back to the dashboard, the `setInterval` keep on running.